Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigationItem.m

```diff
@@ -111,8 +127,55 @@ - (UIColor *)contentColor {
     return BPKColor.gray300;
 }
 
+- (void)setIconColor {
+    if (self.iconView != nil) {
+        self.iconView.tintColor = [self contentColor];
+    }
+}
+
+- (void)updateIconStyle {
+    if (self.iconView != nil) {
+        [self.iconView removeFromSuperview];
+    }
+
+    if (self.definition.iconName == nil) {
+        [self setContentHorizontalAlignment:UIControlContentHorizontalAlignmentCenter];
+        return;
+    }
+
+    BPKIconSize size = self.size == BPKHorizontalNavigationSizeDefault ? BPKIconSizeLarge : BPKIconSizeSmall;
+    self.iconView = [[BPKIconView alloc] initWithIconName:self.definition.iconName size:size];
+
+    self.iconView.translatesAutoresizingMaskIntoConstraints = NO;
+    self.titleLabel.translatesAutoresizingMaskIntoConstraints = NO;
+
+    [self addSubview:self.iconView];
+
+    [NSLayoutConstraint activateConstraints:@[
```

Comment:
`UIButton` natively supports showing images I think we should prefer this to modifying its view hierarchy